FogCreek/Glitch-Community
# Fluffy piccolo: fix "Unfeature Project" icon

depends on https://github.com/FogCreek/shared-components/pull/58

## Links
* Remix link: https://fluffy-piccolo.glitch.me
* Issue-tracker link: https://app.zenhub.com/workspaces/community-5d07d1359dee6059a9688176/issues/fogcreek/glitch-community-work/514

## GIF/Screenshots:
![Screen Shot 2019-10-15 at 10 57 53 AM](https://user-images.githubusercontent.com/938722/66843509-e45db980-ef3a-11e9-85d1-0b4295a662fe.png)

## Changes:
* replace skinny arrow with block arrow

## How To Test:
* log into https://fluffy-piccolo.glitch.me
* visit your profile page
* feature a project
* click on the featured project options menu
* the unfeature project icon should be a block arrow emoji

## Things left to do before deploying:
- [ ] update to production release of shared components, after that's merged
